opennms-doc/guide-development/src/asciidoc/text/rest/situation-feedback.adoc

```diff
@@ -0,0 +1,100 @@
+
+[[gs-situation-feedback]]
+
+==== Situation Feedback
+
+Situation Feedback is an optional feature that allows for the management of User Contributed Feedback regarding Correlation of Alarms.
+
+It is a simple API that allows for the submission and retrieval of Correlation Feedback.
+
+===== GETs (Reading Data)
+
+[options="header", cols="1,10"]
+|===
+| Parameter                 | Description
+| `/situation-feedback/{situationId}` | Get Situation Feedback for the Situation with AlarmId = {situationId}.v
```

Comment:
Trailing `.v`?

Reply:
Might be worth adding a note here that the "situationId" is the same as the alarm id for the situation.